helm-charts/splunk-otel-collector/templates/config/_otel-agent.tpl

```diff
@@ -405,7 +405,14 @@ receivers:
   {{- end }}
 
   {{- if .Values.logsCollection.extraFileLogs }}
-  {{- toYaml .Values.logsCollection.extraFileLogs | nindent 2 }}
+  {{- $extraFileLogsList := .Values.logsCollection.extraFileLogs }}
+  {{- range $extraFileLogKey, $extraFileLogValue := $extraFileLogsList }}
+  {{- printf "%s:" $extraFileLogKey | nindent 2 }}
+  {{- if not $extraFileLogValue.storage }}
+    storage: file_storage
```

Comment:
I'm curios how this works with 2 spaces indent. I believe it should be 4, no?

Reply:
Actually there are 4 spaces before `storage: file_storage`. I matched the indentation of the other elements of the filelog receiver.